### PR TITLE
DeliveriesBackend styled + test

### PR DESCRIPTION
**Pull request by JeremybellEU**
Styled and tested DeliveriesBackend. Also added 1 more test to CourseBackendTest.

### CODE REVIEW

File: src/test/java/nl/tudelft/ewi/devhub/server/backend/DeliveriesBackendTest.java [hunk 1]
```diff
@@ -0,0 +1,365 @@
...
+	public void attachFileToDelivery() throws UnauthorizedException, ApiError {
+		attachedFile();
+	}
+
+	private void attachedFile() throws ApiError {
+		this.deliveriesBackend.attach(delivery, fileName, in);
+		
+		assertEquals(1, delivery.getAttachments().size());
+		// Because a simple get did not return the correct deliveryAttachment, we had to do it the fancy way.
+		assertEquals(1, delivery.getAttachments().stream()
```
Comment (JWGmeligMeyling) [inline src/test/java/nl/tudelft/ewi/devhub/server/backend/DeliveriesBackendTest.java]: assertEquals(fileName, delivery.getAttachments().iterator().next().getFileName()) did not do the trick?

File: src/test/java/nl/tudelft/ewi/devhub/server/backend/DeliveriesBackendTest.java [hunk 2]
```diff
@@ -0,0 +1,365 @@
...
+import static org.mockito.Matchers.any;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+
+@RunWith(MockitoJUnitRunner.class)
+public class DeliveriesBackendTest extends BackendTest {
+
+	private static final String fileName = "myFile.txt";
+
+	private static final String pathName = "my/fancy/path";
```
Comment (JWGmeligMeyling) [inline src/test/java/nl/tudelft/ewi/devhub/server/backend/DeliveriesBackendTest.java]: should end with / as it is a folder

**Comment by JWGmeligMeyling** [comment]
```

**Comment by JeremybellEU** [comment]
Output on local system when changing to fileName instead of FULL_PATH_NAME:
```
Failed tests:   attachFileToDelivery(nl.tudelft.ewi.devhub.server.backend.DeliveriesBackendTest): expected:<my[]File.txt> but was:<my[/fancy/path/my]File.txt>
  adminCanAttachDelivery(nl.tudelft.ewi.devhub.server.backend.DeliveriesBackendTest): expected:<my[]File.txt> but was:<my[/fancy/path/my]File.txt>
  groupMemberCanAttachDelivery(nl.tudelft.ewi.devhub.server.backend.DeliveriesBackendTest): expected:<my[]File.txt> but was:<my[/fancy/path/my]File.txt>
  assistantCanAttachDelivery(nl.tudelft.ewi.devhub.server.backend.DeliveriesBackendTest): expected:<my[]File.txt> but was:<my[/fancy/path/my]File.txt>
```

File: src/test/java/nl/tudelft/ewi/devhub/server/backend/DeliveriesBackendTest.java [hunk 3]
```diff
@@ -0,0 +1,360 @@
...
+import static org.junit.Assert.assertNotNull;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+
+@RunWith(MockitoJUnitRunner.class)
+public class DeliveriesBackendTest extends BackendTest {
+
+	private static final String fileName = "myFile.txt";
+
+	private static final String pathName = "my/fancy/path/";
```
Comment (JWGmeligMeyling) [inline src/test/java/nl/tudelft/ewi/devhub/server/backend/DeliveriesBackendTest.java]: Path should be separated by the path separator used by the operating system. Therefore the test fails in Linux (CI) environments. Use: `"my/fancy/path/".replaceAll("/", File.separator)`.

**Comment by JWGmeligMeyling** [comment]
Merged per https://github.com/devhub-tud/devhub/commit/6ee06f64c36b7f6c417429b931095634d0dc20b0

**Pull request closed without merging**